lib/CompilerInterfaceDevice.h

```diff
@@ -49,9 +67,16 @@ static inline auto __jit_launch_kernel_internal(
 
   TIMESCOPE("__jit_launch_kernel");
   DBG(printKernelLaunchInfo());
-  auto &Jit = JitDeviceImplT::instance();
+
+  SmallVector<RuntimeConstant> RCs;
+  for (size_t Idx = 0; Idx < NumRuntimeConstants; ++Idx) {
+    auto RC = RuntimeConstant {.PtrVal = KernelArgs[RCIndices[Idx]] };
+    RCs.push_back(RC);
+  }
```

Comment:
Why copy again? We should be able to just pass the data

Reply:
Latest commit changes how runtime constants are handled.  Closing